platforms/core-runtime/stdlib-java-extensions/src/main/java/org/gradle/internal/service/scopes/Scope.java

```diff
@@ -173,14 +174,30 @@ interface BuildTree extends BuildSession {}
     interface Build extends BuildTree {}
 
     /**
-     * The scope of the {@code org.gradle.api.invocation.Gradle} instance state.
+     * The scope of the {@code org.gradle.api.invocation.Gradle} instance of a build.
+     * <p>
+     * The state and the services are created at the same time the {@code Gradle} instance
+     * is initialized for a given <em>build</em>, and they are discarded at the end of the build execution.
      * <p>
      * The <em>Gradle state</em> is being merged into the {@link Build build state} and is mostly empty.
      * <p>
      * {@link Build} and parent services are visible to {@link Gradle} services and descendant scopes, but not vice versa.
      */
     interface Gradle extends Build {}
```

Comment:
:question: Didn't Adam remove the `Gradle` scope for good a while ago?

Reply:
Unfortunately, not. It's almost empty, but not entirely.

Reply:
Here are the main services that currently belong to this scope. I'm not sure if there are more that are unaccounted for.

https://github.com/gradle/gradle/blob/35119a2bbcd92b85be50dcd53e004e24161cdfe8/subprojects/core/src/main/java/org/gradle/internal/service/scopes/GradleScopeServices.java#L79